src/main/java/de/craften/plugins/rpgplus/RpgPlus.java

```diff
@@ -60,7 +63,14 @@ public RpgPlus() {
         timerManager = new TimerComponent();
         
         StorageType storageType = StorageType.valueOf(getConfig().getString("storageType", "YAML").toUpperCase());
-        storage = new StorageComponent(storageType, new File(getDataFolder(), "storage"));
+        
+        if (storageType == StorageType.MEMORY) {

```

Comment:
The `StorageType` enum can now be removed. Just check for string equality here. For all users of the storage module, the underlying storage engine is an implementation detail that should be hidden.